config/initializers/console.rb

```diff
@@ -22,13 +40,42 @@ def show_warning_message_about_environments
       puts '** You are in the Rails console for PRODUCTION! **'.red
       puts ('*' * 50).red
     else
-      puts ('-' * 65).blue
-      puts "-- This is the Rails console for the #{HostingEnvironment.environment_name} environment. --".blue
-      puts ('-' * 65).blue
+      puts ('-' * 65).light_blue
+      puts "-- This is the Rails console for the #{HostingEnvironment.environment_name} environment. --".light_blue
+      puts ('-' * 65).light_blue
```

Comment:
I changed to light blue because I have eye sight problem and light blue is easier to identify.

Reply:
Btw, I wish we could generate different phrases for the console 😁😁😁😁😁😁😁😁 (I won't do on this PR though!)